packages/adapter/src/methods.ts

```diff
@@ -47,54 +47,51 @@ export async function isPolkadotSnapInstalled(pluginOrigin: string): Promise<boo
 
 export async function addKusamaAsset(pluginOrigin: string): Promise<void> {
   await window.ethereum.send({
-      method: pluginOrigin,
-      params: [{
-          method: 'addKusamaAsset'
-      }]
+    method: pluginOrigin,
+    params: [{
+      method: 'addKusamaAsset'
+    }]
   });
 }
```

Comment:
You should delete this whole function as it got replaced with `addPolkadotAsset`